kubectl-rook-ceph.sh

```diff
@@ -493,6 +493,42 @@ function run_debug() {
     ;;
   esac
 }
+
+function run_dr_clusters_health() {
+  [[ -z "${1:-""}" ]] && fail_error "Missing subcommand"
+  subcommand="$1"
+  shift # remove the subcommand from the front of the arg list
+  case $subcommand in
+  health)
+    run_dr_health "$@"
+    ;;
+  *)
+    fail_error "'dr' subcommand '$subcommand' does not exist"
+    ;;
+  esac
+}
+
+function run_dr_health() {
+  # do not call end_of_command_parsing here because all remaining input is passed directly to 'ceph'
+  info_msg "list of all the blockpools: "
+  KUBECTL_NS_CLUSTER get cephblockpool | awk '{print $1}' | sed "1 d"
+  echo
+  info_msg "enter ceph cephblockpool name: "
```

Comment:
why this is configurable? just list the pools and check if is it mirroring enabled and use secret from it instead of asking the user for input.

Reply:
nice idea

Reply:
done